### PR TITLE
New SRF API / LiveTV with DRM / Subtitles 

### PR DESCRIPTION
**Pull request by tobiaswaldvogel**
This is major rework in order to implement:

- New SRF API
- LiveTV with DRM 
- Radio channels
- Use streaminput adaptive in order to adapt dynamically to the internet speed and support DRM
- Subtitle support

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.srgssr" name="SRG SSR" version="1.7.0" provider-name="Alexander Seiler">
+<addon id="script.module.srgssr" name="SRG SSR" version="2.0.0" provider-name="Alexander Seiler">
   <requires>
-    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="xbmc.python" version="3.0.0"/>
```
Comment (goggle) [inline addon.xml:4]: Could you revert this change?
It will break all older Kodi versions.

**Comment by tobiaswaldvogel** [comment]
> Would it be also possible to remove all the live TV (DRM) changes from this pull request?
> Live TV is a nice feature, but we should first get `plugin.video.srfplaytv` back working, before adding new features.
> 
> After that, I will merge this and we can polish it directly on the master branch.

No problem, I will create revert and remove the DRM for now.
Afterwards I will open a new PR
